src/handlers/secret.rs

```diff
@@ -0,0 +1,182 @@
+use log::debug;
+use rocket::http::Status;
+use rocket::response::status;
+use rocket::serde::json::Json;
+use rocket::{Response, State};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use serde_json::Value;
+use uuid::Uuid;
+use crate::rocket::futures::TryStreamExt;
+
+use crate::Storage;
+use crate::{
+    error::AppError,
+    storage::{
+        secret::{NewSecret, SecretEntity},
+        storage::AttemptCountRule,
+    },
+};
+
+#[derive(Serialize)]
+pub struct AllSecretsResponse {
+    secrets: Vec<Secret>,
+}
+
+#[derive(Clone, Deserialize)]
+pub struct GuessSecret {
```

Comment:
Just o keep consistency, can we rename this struct to `GuessSecretRequest`?

Reply:
I agree